### PR TITLE
[60] Added logging for all tests except for /info

### PR DESCRIPTION
**Pull request by sam-i-am012**
Addresses #60 
Once PR #69 gets added, then all tests will have logging

### CODE REVIEW

**Comment by sam-i-am012** [comment]
I've rebased onto main to remove the conflicts that would have occurred when rebasing.

File: src/test/java/org/irods/Operations/RuleOperationsTest.java [hunk 1]
```diff
@@ -20,7 +22,7 @@ public class RuleOperationsTest {
     private String rodsToken;
 
     private final ObjectMapper mapper = new ObjectMapper();
-    // private static final Logger logger = LogManager.getLogger(QueryOperationsTest.class);
+     private static final Logger logger = LogManager.getLogger(RuleOperationsTest.class);
```
Comment (korydraughn) [inline src/test/java/org/irods/Operations/RuleOperationsTest.java:25]: Indentation off by one.

File: src/test/java/org/irods/Operations/TicketOperationsTest.java [hunk 2]
```diff
@@ -23,7 +25,7 @@ public class TicketOperationsTest {
 
     private String host;
     private final ObjectMapper mapper = new ObjectMapper();
-    // private static final Logger logger = LogManager.getLogger(TicketOperationsTest.class);
+     private static final Logger logger = LogManager.getLogger(TicketOperationsTest.class);
```
Comment (korydraughn) [inline src/test/java/org/irods/Operations/TicketOperationsTest.java:28]: Indentation.

**Review by korydraughn — COMMENTED** [review]
Please squash this down to one commit. The rebase doesn't need to have a separate commit.

File: src/test/java/org/irods/Operations/RuleOperationsTest.java (hunk 1 above)
Comment (sam-i-am012) [reply src/test/java/org/irods/Operations/RuleOperationsTest.java:25]: Done.

File: src/test/java/org/irods/Operations/TicketOperationsTest.java (hunk 2 above)
Comment (sam-i-am012) [reply src/test/java/org/irods/Operations/TicketOperationsTest.java:28]: Done.

**Comment by sam-i-am012** [comment]
Done.

**Review by korydraughn — APPROVED** [review]
Looks good.

If all the tests pass, pound it.

**Comment by sam-i-am012** [comment]
Yes, tests all pass. I've pounded it.

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**